tests/internal/test_cephfilesystem_creation.py

```diff
@@ -0,0 +1,61 @@
+import logging
+
+from ocs_ci.helpers.helpers import (
+    create_ceph_file_system,
+)
+from ocs_ci.ocs.exceptions import CommandFailed
+import ocs_ci.ocs.resources.pod as pod
+from ocs_ci.framework.testlib import ManageTest
+from ocs_ci.framework.pytest_customization.marks import (
+    tier2,
+    green_squad,
+)
+
+logger = logging.getLogger(__name__)
+
+
+class TestCephFileSystemCreation(ManageTest):
+    """
+    Testing Creation of a filesystem and checking its existence.
+    Also checking if the same filesystem can't be created twice.
+    """
+
+    @tier2
+    @green_squad
```

Comment:
You need to add this TC in polarion and update the polarion ID here.

Reply:
added polarian id